mozjs-sys/mozjs/js/public/CallArgs.h

```diff
@@ -210,7 +210,7 @@ class MOZ_STACK_CLASS MOZ_NON_PARAM CallArgsBase {
 
   /* Returns the i-th zero-indexed argument. */
   MutableHandleValue operator[](unsigned i) const {
-    MOZ_ASSERT(i < argc_);
+    MOZ_RELEASE_ASSERT(i < argc_);
```

Comment:
We hit this in servo: https://github.com/sagudev/servo/actions/runs/19205812345/attempts/1#summary-54902463171

upstream patch: https://hg-edge.mozilla.org/releases/mozilla-esr140/rev/c6c83536941e